### PR TITLE
2.3.92: ability to hasten a Chrome extension update

### PR DESCRIPTION
**Pull request by 2is10**

### CODE REVIEW

File: packrat/adapters/chrome/api.js [hunk 1]
```diff
@@ -63,6 +63,14 @@ api = function() {
     }
   });
 
+  var updateCheckRequested;
+  chrome.runtime.onUpdateAvailable.addListener(function(details) {
+    api.log("#666", "[onUpdateAvailable]", details.version);
```
Comment (eishay) [inline packrat/adapters/chrome/api.js:68]: does this number means anything?

File: packrat/main.js [hunk 2]
```diff
@@ -124,6 +124,12 @@ const socketHandlers = {
     session = null;
     clearDataCache();
   },
+  version: function(v) {
+    api.log("[socket:version]", v);
+    if (api.version != v) {
+      api.requestUpdateCheck();
```
Comment (eishay) [inline packrat/main.js:130]: cool!

File: packrat/adapters/chrome/api.js (hunk 1 above)
Comment (2is10) [inline packrat/adapters/chrome/api.js:68]: gray (a hex color)

**Comment by 2is10** [comment]
added server stuff (admin panel)

**Comment by eishay** [comment]
nice!

File: packrat/adapters/chrome/api.js (hunk 1 above)
Comment (gmethvin) [inline packrat/adapters/chrome/api.js:68]: See also: http://en.wikipedia.org/wiki/666_(number)

**Pull request merged**